modules/030-cloud-provider-yandex/templates/cloud-controller-manager/deployment.yaml

```diff
@@ -126,6 +126,8 @@ spec:
             value: {{ .Values.cloudProviderYandex.internal.providerClusterConfiguration.provider.folderID | quote }}
           - name: YANDEX_CLOUD_DEFAULT_LB_LISTENER_SUBNET_ID
             value: {{ .Values.cloudProviderYandex.internal.defaultLbListenerSubnetId | quote }}
+          - name: YANDEX_CLOUD_DEFAULT_INTERNAL_LB_LISTENER_SUBNET_ID
+            value: {{ index .Values.cloudProviderYandex.internal.providerDiscoveryData.zoneToSubnetIdMap (.Values.cloudProviderYandex.internal.providerDiscoveryData.zoneToSubnetIdMap | keys | first) | quote }}
```

Comment:
why do we use first network?

Reply:
No reason. If I remember correctly, then it makes no difference which one to choose.

Reply:
Balancers are regional, so it makes no difference which zone to choose. Proofs are available in our slack.